GodotEnv/src/features/godot/models/GodotEnvironment.cs

```diff
@@ -115,7 +115,7 @@ bool isTemplate
 public abstract class GodotEnvironment : IGodotEnvironment {
   public const string GODOT_FILENAME_PREFIX = "Godot_v";
   public const string GODOT_URL_PREFIX =
-    "https://downloads.tuxfamily.org/godotengine/";
+    "https://github.com/godotengine/godot/releases/download/";
```

Comment:
I think this URL should be:
```
https://github.com/godotengine/godot-builds/releases/download/
```
Because that is the prefix of the URL provided in the latest 4.2-beta1 announcement, and the URL you have above does not include 4.2-beta1.

Reply:
For more context, observe differences between these two pages:
[https://github.com/godotengine/godot-builds/releases/](https://github.com/godotengine/godot-builds/releases/)
- 4.2-beta1
- 4.1.2-stable
- 4.2-dev6
- etc.

[https://github.com/godotengine/godot/releases](https://github.com/godotengine/godot/releases)
- 4.1.2-stable
- 3.5.3-stable
- 4.0.4-stable
- etc.

It is particularly frustrating that there isn't a single URL that provides all of the releases, since I'm most interested in downloading 4.2-beta1.

Would it be a reasonable refactor to support querying both URLs to see whether either offers the requested version?

Reply:
Upon further review, it does look like `godot-builds/releases` may contain a strict superset of the releases available from `godot/releases`